features/unit/v2algodclient.feature

```diff
@@ -0,0 +1,76 @@
+Feature: Algod REST API v2
+
+  Scenario Template: Shutdown
+    When we make any Shutdown call, return mock response "<jsonfile>" and expect error string to contain "<err>"
+    Examples:
+      |jsonfile|err|
+      |TODO    |nil|
+
+  Scenario Template: Register Participation Keys
+    When we make any Register Participation Keys call, return mock response "<jsonfile>" and expect error string to contain "<err>"
+    Examples:
+      |jsonfile|err|
+      |TODO    |nil|
+
+  Scenario Template: Pending Transaction Information
+    When we make any Pending Transaction Information call, return mock response "<jsonfile>" and expect error string to contain "<err>"
+    Then the parsed Pending Transaction Information response should have sender "<sender>"
+    Examples:
+      |jsonfile|err|sender|
+      |TODO    |nil|TODO  |
```

Comment:
Something a lot of these tests do is only check one field. I wonder if there's a way we can check every field, or at least more fields, without the tests getting really verbose and niche.